### ISSUE TITLE
IMOS:  check_coordinate_variables - float variable is not seen as numeric

### ISSUE THREAD

**Issue by lbesnard** [aodn/compliance-checker#17]
The IMOS checker complains that the HEIGHT_ABOVE_SENSOR variable which is a float is not numeric. However the definition is 

``` python
    def check_coordinate_variables(self, ds):
        """
        Check all coordinate variables to ensure it has numeric type (byte,
        float and integer) and also check whether it is monotonic
        """
```

netcdf-checker output

```
var                                    :3:   695/715 :
    HEIGHT_ABOVE_SENSOR                :3:    10/11 :
        check_variable_type            :3:     0/ 1 : Variable type is not
                                                      numeric
```

ncdump

```
float HEIGHT_ABOVE_SENSOR(HEIGHT_ABOVE_SENSOR) ;
...
HEIGHT_ABOVE_SENSOR = 23.17, 38.17 ...
```

Example with file:
/mnt/ebs/wip/ANMN/timeSeriesProfile_po_permission/new_tb/IMOS_ANMN-QLD_AETVZ_20130620T075900Z_ITFTIS_FV01_ITFTIS-1306-Workhorse-ADCP-464.5_END-20140111T055900Z_C-20150515T020357Z.nc

(which has also an exception)

**Update**
Similar with variable `DIST_ALONG_BEAMS` from /mnt/ebs/wip/ANMN/timeSeriesProfile_po_permission/new_tb/IMOS_ANMN-QLD_AETVZ_20130620T075902Z_ITFTIS_FV00_ITFTIS-1306-Workhorse-ADCP-464.5_END-20140111T055902Z_C-20150515T020357Z.nc

**Comment by fxmzb123** [comment]
@lbesnard and @mhidas Can you make the example nc file available through thread server?

**Comment by fxmzb123** [comment]
@lbesnard and @mhidas i have fixed this and created a pull request. Please test it.

**Comment by mhidas** [comment]
@fxmzb123 Looks good, thanks. 

Fixed in #27
